_scripts/all_pages.rb

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env ruby
+
+require 'octokit'
+
+TOKEN_PATH = File.join(ENV['HOME'], '.github_token')
+
+unless File.exist? TOKEN_PATH
+  $stderr.puts "No GitHub token found at #{TOKEN_PATH}"
+  exit 1
+end
+
+access_token = File.read TOKEN_PATH
+Octokit.auto_paginate = true
+client = Octokit::Client.new access_token: access_token
+repos = client.org_repos '18F'
+
+pages_repos = repos.map do |repo|
```

Comment:
This could be written a lot more functionally, if you want. :)

```ruby
repos
  .collect { |repo| repo.full_name }
  .select { |name| client.branches(name).any? { |b| b.name == '18f-pages' } }
  .sort
  .map { |repo| puts repo }
```